lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -48,11 +51,14 @@ class LoadFastEnough4Pwa extends Audit {
     // If throttling was default devtools or lantern slow 4G throttling, then reuse the given settings
     // Otherwise, we'll force the usage of lantern slow 4G.
     const settingOverrides = {throttlingMethod: 'simulate', throttling: mobileThrottling};
-    const settings =
-      context.settings.throttlingMethod !== 'provided' &&
-      isDeepEqual(context.settings.throttling, mobileThrottling)
-        ? context.settings
-        : Object.assign({}, context.settings, settingOverrides);
+
+    // Override settings for interactive if provided throttling was used and network
+    // throttling was not consistent with standard `mobile network throttling`
+    const override = context.settings.throttlingMethod === 'provided' &&
+    !isDeepEqual(context.settings.throttling, mobileThrottling);
+
+    const settings = !override? context.settings:
```

Comment:
nit: spacing around `?`, also for the ternary I usually find it easier if there's not another layer of negation, mind switching the order and removing `!`?